woocommerce/woocommerce
# Update automation bypass to work correctly, consider actions bot.

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates the bypass for checks that run due to automation to function correctly. PRs created by the github-actions bot will have their requirements satisfied. This also updates the checks for non-relevant files to move all the skips within a single file.



Closes #36659.



### How to test the changes in this Pull Request:




**For all of the following steps, be very careful that you're acting in your fork, and not creating releases, milestones, etc in the monorepo**

1. Fork this repository. While forking, it's helpful to copy all branches so that you get release branches as well. Ensure you have a `release/7.4` branch.
1. Enable actions in your fork, and make sure that workflow permissions are set to "Read and write permissions", and "Allow GitHub Actions to create and approve pull requests" is checked (`/settings/actions`).
1. Make sure that issuses are enabled in your fork (`/settings`).
1. Create numbered milestones `7.4.0`, and `7.5.0` in milestones (`/milestones`).
1. Create a `7.4.0` release in your fork which tags the 
1. Add required checks in your fork's `trunk` branch that match the required checks for `trunk` in the monorepo (`/settings/branches`).
1. Apply the changes in this PR to `trunk` in your fork.
1. Create a new PR in your fork with milestone 7.5.0 **Verify that the required checks run as intended, and that you are able to merge the PR without bypassing the checks.**
1. Merge the PR. The automation should place it in the `7.5.0` milestone.
1. Edit the milestone to `7.4.0`, this should trigger automation to cherry-pick the PR into the 7.4 release branch.
1. When the PR is created deleting the changelog, close and reopen it to trigger the running of checks.
1. Verify that after the skip checks are run, you're able to merge the PR without bypassing the checks.




### Other information:




### FOR PR REVIEWER ONLY:
